x-pack/test/apm_api_integration/tests/mobile/mobile_http_requests_timeseries.spec.ts

```diff
@@ -75,25 +75,25 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
         expect(response.status).to.be(200);
         expect(
-          response.body.currentPeriod.some(
+          response.body.currentPeriod.timeseries.some(
             (item: { x: number; y?: number | null }) => item.y === 0 && item.x
```

Comment:
Is the explicit type needed? I'd think not?

```suggestion
            (item) => item.y === 0 && item.x
```